pkg/userdefinedinjections/userdefinedinjections.go

```diff
@@ -0,0 +1,112 @@
+package userdefinedinjections
+
+import (
+	"encoding/json"
+	"reflect"
+	"strings"
+	"sync"
+
+	"github.com/golang/glog"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/k8snetworkplumbingwg/network-resources-injector/pkg/types"
+)
+
+const (
+	userDefinedInjectionsMainKey = "user-defined-injections"
+)
+
+// UserDefinedInjections user defined injections
+type UserDefinedInjections struct {
+	sync.Mutex
+	Patchs map[string]types.JsonPatchOperation
+}
+
+// CreateUserInjectionsStructure returns empty UserDefinedInjections structure
+func CreateUserInjectionsStructure() *UserDefinedInjections {
+	var userDefinedInjects = UserDefinedInjections{Patchs: make(map[string]types.JsonPatchOperation)}
+	return &userDefinedInjects
+}
+
+// SetCustomizedInjections sets additional injections to be applied in Pod spec
+func (userDefinedInjects *UserDefinedInjections) SetCustomizedInjections(injectionsCm *corev1.ConfigMap) {
```

Comment:
SetCustomizedInjections -> SetUserDefinedInjections

Reply:
Done.

I should do a training from search tool ;) - how could I missed that. Sorry for double work on review.

Reply:
Done